src/features/bondfeatures.jl

```diff
@@ -0,0 +1,18 @@
+include("abstractfeatures.jl")
+
+abstract type BondFeature <: AbstractPairFeature end
+
+#=
+TODO - Come up with a better name for these type
```

Comment:
usually the term is "bond order" I think

Reply:
Yes, but for some, the Bond Order isn't defined to be only 1, 2, or 3. 
For instance, in `N=0`, the Lewis structure suggests a double bond, but the canonical bond order is 2.5 (which we don't explicitly support). Which is why I felt it might be a little confusing to name it `BondOrder` and settled with this .

Reply:
hmmm fair point #ntih

Reply:
Probably worth mentioning in a docstring why we chose to name it this way too